internal/api/scim/integration_test/users_replace_test.go

```diff
@@ -303,7 +303,14 @@ func TestReplaceUser_removeOldMetadata(t *testing.T) {
 			Id: createdUser.ID,
 		})
 		require.NoError(tt, err)
-		require.Equal(tt, 0, len(md.Result))
+		require.Equal(tt, 1, len(md.Result))
+
+		mdMap := make(map[string]string)
+		for i := range md.Result {
+			mdMap[md.Result[i].Key] = string(md.Result[i].Value)
+		}
+
+		test.AssertMapContains(tt, mdMap, "urn:zitadel:scim:emails", "[{\"value\":\"user1@example.com\",\"primary\":true}]")
```

Comment:
Somewhat unsure about this test, but I think it makes sense as emails are now stored in the metadata. Feel free to correct me if I'm wrong

Reply:
I believe the email type is also stored in the metadata, see for example https://github.com/nais/zitadel/blob/scim-user-emails-type/internal/api/scim/integration_test/users_create_test.go#L394

I've added an explicit test for this below

Reply:
True, might have missed that on my side, works out for me.